kodata/events/toc-review.rotation

```diff
@@ -0,0 +1,18 @@
+#@ title: ToC Working Group Update
+#@ duration: 45m
+
+# Start | Group
+2021-03-11T08:30:00-07:00 | Security WG
```

Comment:
why is this `-07:00 ` and the others are `:08:00` ? do we need the end date if we have the duration?

Reply:
This is a timezone offset in RFC3339. The others are `17:00:00Z` (Zulu), which is 9AM Pacific in the winter, and 10AM in the summer.

Of course, time zones suck enough that I got this one backwards, and it should be -07:00 next week, and -08:00 now.

Reply:
RFC3339 doesn't support "PST"/"PDT" offsets, but I could be convinced to change the time format...